include/drivers/can.h

```diff
@@ -1239,6 +1185,61 @@ __deprecated static inline void can_register_state_change_isr(const struct devic
 	can_set_state_change_callback(dev, isr, NULL);
 }
 
+/**
+ * @brief Wrapper function for writing data to the CAN bus.
+ *
+ * Simple wrapper function for @a can_send() without the need for filling in a
+ * @a zcan_frame struct. This function blocks until the data is sent or a
+ * timeout occurs.
+ *
+ * By default, the CAN controller will automatically retry transmission in case
+ * of lost bus arbitration or missing acknowledge. Some CAN controllers support
+ * disabling automatic retransmissions ("one-shot" mode) via a devicetree
+ * property.
+ *
+ * @warning This function is deprecated. Use @a can_send() instead.
```

Comment:
```suggestion
 * @deprecated Use can_send() instead.
```

so it appears here https://docs.zephyrproject.org/latest/doxygen/html/deprecated.html

Reply:
Done. However, the deprecated CAN API functions are currently excluded from the documentation (the CAN API is marked as unstable and has seen so many changes for v3.0 that the documentation would be unreadable if all the deprecated functions where mixed in).